orbit/cmd/orbit/orbit.go

```diff
@@ -1514,6 +1519,60 @@ func main() {
 
 		go sigusrListener(c.String("root-dir"))
 
+		isLinux := runtime.GOOS == "linux"
+		serverHasWebSetup := orbitClient.GetServerCapabilities().Has(fleet.CapabilityWebSetupExperience)
+		setupExperienceNotDisabled := !c.Bool("disable-setup-experience")
+		runSetupExperience := isLinux && serverHasWebSetup && setupExperienceNotDisabled
+		log.Debug().
+			Bool("isLinux", isLinux).
+			Bool("serverHasSetup", serverHasWebSetup).
+			Bool("notDisabled", setupExperienceNotDisabled).
+			Msg("checking setup experience preflight values")
+
+		openMyDevicePage := func() error {
+			log.Debug().Msg("launching browser for my device page")
+			token, err := trw.Read()
+			if err != nil {
+				return fmt.Errorf("getting device token: %w", err)
+			}
+			// My Device page
+			browserURL := deviceClient.BrowserDeviceURL(token)
```

Comment:
Should we wait until the token and URL are valid (return 200) before attempting to open with the browser?